extensions/amp-form/0.1/amp-form.js

```diff
@@ -556,14 +556,14 @@ export class AmpForm {
       // Validity checking should always occur, novalidate only circumvent
       // reporting and blocking submission on non-valid forms.
       const isValid = checkUserValidityOnSubmission(this.form_);
-      if (this.shouldValidate_ && !isValid) {
+      if (this.shouldValidate_) {
```

Comment:
Does this change behavior? It will cause `reportValidity` to be called which I think could have side effects even if there are no changes. Disregard if you already verified.